src/main/java/uk/gov/hmcts/reform/pip/rules/errorhandling/GlobalExceptionHandler.java

```diff
@@ -15,15 +17,16 @@
 @ControllerAdvice
 public class GlobalExceptionHandler {
 
+
     /**
-     * Template exception handler, that handles a custom PublicationNotFoundException,
+     * Template exception handler, that handles a custom CourtNotFoundException,
      * and returns a 404 in the standard format.
      * @param ex The exception that has been thrown.
      * @return The error response, modelled using the ExceptionResponse object.
      */
-    @ExceptionHandler(PublicationNotFoundException.class)
-    public ResponseEntity<ExceptionResponse> handlePublicationNotFound(
-        PublicationNotFoundException ex) {
+    @ExceptionHandler(CourtNotFoundException.class)
```

Comment:
we still have publication not found exception and it doesnt look that that would be handled anymore